addons/bus/static/src/js/services/bus_service.js

```diff
@@ -82,6 +82,8 @@ var BusService =  CrossTab.extend(ServicesMixin, {
                 }
             }
         } else {
+            options.message = _.unescape(options.message)
+            options.title = _.unescape(options.title)
```

Comment:
I'm a little bit confused about that we should escape only those messages containing symbols '<, >, &' 
and not all messages? But as a result, when these symbols are escaped, they will be converted to '&lt; &gt; &amp;'.

And if I am understanding anything wrong, can you please help me clear out things?

Reply:
We should know exactly where the message is escaped, so that we know for sure why it was escaped and we know it's not a security risk. Here by unescaping all messages, some messages might not have been escaped, so it's a security issue.

I would even say that using `_.unescape()` shows a problem in the code. Because if you need the unescaped version, then we shouldn't have to escape it in the 1st place (or there should be a way to get the unescaped version, without needing to make this transformation on the fly).

I much prefer we improve / fix this issue in `master`, and consider doing that in stable if we are sure we can put `escape()` at the right places. If we cannot easily do that in stable, I much prefer keeping this bug of showing escaped content than have a security issue.